week3/homework/src/index.js

```diff
@@ -1,3 +1,88 @@
 'use strict';
 
-// TODO: Write the homework code in this file
+const Express = require('express');
+
+// import our CRUD actions
+const { createTodo, readTodos, updateTodo, deleteTodo } = require('./actions');
+
+const Todo = require('./todo');
+
+const FILENAME = 'todos.json';
+const PORT = 3000;
+const TODO_SLUG = 'todos';
+
+const todo = new Todo(FILENAME);
+
+const app = new Express();
+
+// Use built-in JSON middleware to automatically parse JSON
+app.use(Express.json());
+
+// Functions that I created
+function readTodo(todo, request, response) {
+  todo.read().then(todos => {
+    const todoId = request.params.id;
+    const pickTodo = todos.find(todo => todo.id === todoId);
```

Comment:
nice, finally someone who does not abuse map or filter to find a single element 🥇